README.md

```diff
@@ -1,4 +1,9 @@
 nfc
 ===
 
-on behalf of W3C NFC working group
\ No newline at end of file
+on behalf of W3C NFC working group
+
+index.html: the current Editor's draft with separate handling of peers and tags.
+
+index2.html: an experimental version which merges peer and tag read handling,
```

Comment:
On February 2, 2015 at 10:44:06 PM, Zoltan Kis (notifications@github.com) wrote:
> > But we have two entirely different concepts of API here. It cannot  
> be merged into one doc. 

Ok, I'm just worried that people won't see the second proposal. 

> If we have to stick with one, it's the current  
> one, which just got through the CfC. The other then cannot be published  
> yet, there is an issue for tracking it, then that's it. What is  
> the problem in having two documents there?

As above: it's not always clear which one contains the latest stuff. That's why it's always best just to put everything in one place and put markers in the document, like:

"We are currently considering the following two approaches - see issue X and Y on GitHub".


> We do need both experimental  
> and base versions accessible from the web, i.e. they need to be  
> two different files in the same branch.

The above addresses this. However, if you think it's too messy, that's ok. 